src/main/python/opensearchsql_cli/tests/sql/test_sql_version.py

```diff
@@ -144,7 +144,7 @@ def test_get_jar_path(
         Test getting JAR path
         """
         # Setup mock
-        expected_path = "/mock/project_root/build/libs/opensearchsqlcli-3.3.0.0.jar"
```

Comment:
This breaks every time there's a version bump upstream -- longterm fix for versioning is TODO